stable/heapster/templates/deployment.yaml

```diff
@@ -30,6 +30,10 @@ spec:
           timeoutSeconds: 5
         resources:
 {{ toYaml .Values.resources | indent 12 }}
+{{- if .Values.env }}
```

Comment:
I'd like you add the `Values.env` to `values.yaml`, also update the chart version from 0.1.1 to 0.1.2